engines/experiment/plugin/rpc/runner/plugin.go

```diff
@@ -12,10 +14,38 @@ type ExperimentRunnerPlugin struct {
 	Impl ConfigurableExperimentRunner
 }
 
-func (p *ExperimentRunnerPlugin) Server(*plugin.MuxBroker) (interface{}, error) {
-	return &rpcServer{Impl: p.Impl}, nil
+func (p *ExperimentRunnerPlugin) Server(b *plugin.MuxBroker) (interface{}, error) {
+	return &rpcServer{Impl: p.Impl, MuxBroker: b}, nil
```

Comment:
The MuxBroker is added now for the bidirectional call, correct?

Reply:
Yep that's right.